src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(currentNum) {
+  let currentNummount = currentNum || 0;
+
+  function sum(nextNum) {
+    if (nextNum !== undefined) {
+      currentNummount += nextNum;
+
+      return sum;
+    } else {
+      const result = currentNummount;
+
+      currentNummount = 0;
```

Comment:
Resetting `currentNummount` here is incorrect due to the typo. It should be `currentNum` to properly reset the accumulator.